cmd/rollkit/commands/run_node.go

```diff
@@ -359,36 +337,16 @@ func tryStartMockSequencerServerGRPC(listenAddress string, rollupId string) (*gr
 	return server, nil
 }
 
-// tryStartMockExecutorServerGRPC will try and start a mock gRPC executor server
-func tryStartMockExecutorServerGRPC(listenAddress string) (*grpc.Server, error) {
-	dummyExec := execTest.NewDummyExecutor()
-
-	go func() {
-		ticker := time.NewTicker(100 * time.Millisecond)
-		defer ticker.Stop()
-		i := 0
-		for range ticker.C {
-			dummyExec.InjectTx(execTypes.Tx{byte(3*i + 1), byte(3*i + 2), byte(3*i + 3)})
-			i++
-		}
-	}()
+// createDirectKVExecutor creates a KVExecutor for testing
+func createDirectKVExecutor() *testExecutor.KVExecutor {
```

Comment:
awesome job, i think we are still missing the http server in kv executor to be able to submit txs or a cli command to submit them allowing tests to run the complete e2e system